app/controllers/schools/participants_controller.rb

```diff
@@ -96,15 +97,15 @@ def check_feature_flag
   end
 
   def set_participant
-    @participant = User.find(params[:participant_id] || params[:id])
-    authorize @participant, policy_class: ParticipantPolicy
+    @profile = ParticipantProfile.find(params[:participant_id] || params[:id])
```

Comment:
Will that work? what is `participant_id`? I thought it would be the id of the user?

Reply:
I changed it to be the ID of a profile